README.md

```diff
@@ -126,6 +208,19 @@ and others.  Each of those projects has a demo application for specific hardware
 In the case of the ARM7 and AVR, their makefile works with GCC compilers and
 there are project files for IAR Embedded Workbench and Rowley Crossworks for ARM.
 
+Commonly used apps:
```

Comment:
Please reference the [bin/readme.txt](https://github.com/bacnet-stack/bacnet-stack/tree/master/bin) which already has this information.